### PR TITLE
add react-redux-form@1.5.3 w/ npm auto-update

### LINKED ISSUE #9759: [Request] Add react-redux-form
**Library name:** react-redux-form
**Git repository url:** https://github.com/davidkpiano/react-redux-form
**npm package url(optional):** https://www.npmjs.com/package/react-redux-form
**License(s):** MIT
**Official homepage:** https://davidkpiano.github.io/react-redux-form/
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by sufuf3**
close #9759, cc @davidkpiano
@kennynaoh please help me review this PR, thank you.

Pull request for issue: #9759
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/davidkpiano/react-redux-form
 * Official website (optional, not the repository): https://davidkpiano.github.io/react-redux-form/
 * NPM package url (optional): https://www.npmjs.com/package/react-redux-form
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 1043
   - Count of watchers: 34
   - Count of forks: 108
 * NPM download stats (optional):
   - Downloads in the last day: 1036
   - Downloads in the last week: 4603
   - Downloads in the last month: 15638

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/react-redux-form/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  },
+  "license": "MIT",
+  "homepage": "https://davidkpiano.github.io/react-redux-form/",
+  "filename": "ReactReduxForm.min.js",
+  "npmName": "react-redux-form",
+  "npmFileMap": [
+    {
+      "basePath": "umd",
+      "files": [
+        "**/*"
```
Comment (kennynaoh) [inline ajax/libs/react-redux-form/package.json]: @sufuf3 Let's use `ReactReduxForm*` here.

**Comment by sufuf3** [comment]
@kennynaoh please help me review this PR again, thank you.

File: ajax/libs/react-redux-form/package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  },
+  "license": "MIT",
+  "homepage": "https://davidkpiano.github.io/react-redux-form/",
+  "filename": "ReactReduxForm.min.js",
+  "npmName": "react-redux-form",
+  "npmFileMap": [
+    {
+      "basePath": "umd",
+      "files": [
+        "**/ReactReduxForm*"
```
Comment (kennynaoh) [inline ajax/libs/react-redux-form/package.json]: Just `ReactReduxForm*` here.

**Comment by sufuf3** [comment]
@kennynaoh please help me review this PR again, thank you.

File: ajax/libs/react-redux-form/package.json (hunk 2 above)
Comment (PeterDaveHello) [inline ajax/libs/react-redux-form/package.json]: Actually, I'm fine with both.

**Pull request merged**